src/hint.js

```diff
@@ -0,0 +1,18 @@
+import flag from 'cozy-flags'
+
+const fetchHint = async client => {
+  if (flag('passwords.force-no-hint')) {
+    return false
+  }
+  try {
+    const hint = await client
+      .getStackClient()
+      .collection('io.cozy.settings')
+      .get('hint')
```

Comment:
can't you : 
```
client.query(client.get('io.cozy.settings', 'hint'))
``` 
?

Reply:
No this is not possible as the hint does not return any data (204 No content), and it does not work with `query` in this case. Thank for asking though, I had not checked.

Reply:
https://github.com/cozy/cozy-passwords/pull/25/commits/229f29e3604aa4e2d50396fc2176d63cc69eb78e makes it clearer.